doc/riscv_trace.rst

```diff
@@ -0,0 +1,69 @@
+.. _riscv_trace:                                                                                         
+
+   :name: riscv-trace-spec
+   :align: center
+
+RISC-V Trace Specs Interface
+============================
+
+Ibex implements a trace interface compatible with the riscv-trace-spec Version 1.0 issued in March 20, 2020. 
+This interface currently exports needed internal signal as ibex_core outputs to be used to encode the trace packets to be logged.
+Signals implemented until now are shown in the table below:
+
+
+Trace Signals
+-------------
+
+For now, only single retirement per block is implemented. 
```

Comment:
Ibex doesn't retire more than one instruction per cycle, and it's unlikely that this will ever change (famous last words). So you can remove this sentence.